package.json

```diff
@@ -113,7 +116,16 @@
   },
   "dependencies": {
     "@hyphen/hyphen-design-tokens": "^2.0.2",
+    "@popperjs/core": "^2.11.8",
     "classnames": "^2.5.1",
-    "react-select": "^5.8.0"
+    "cleave.js": "^1.6.0",
```

Comment:
This was causing problems with dependencies on next correct? We (@nathanyoung and I) said that we didn't need to worry about masking for now so we just should take out the masking that depended on this package.

Reply:
Sure, I’ll remove it asap.